.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+
+node_js:
+- 'node'
+
+os:
+- linux
+- osx
+
+before_install:
+- if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+  fi
+
+install:
+- npm install -g vsce
+- npm install -g typescript
+- npm install -g gulp
+- npm install
+
+script:
+- gulp tslint
+- vsce package
+# - npm test --silent

```

Comment:
What would this do if we enabled it?  We do have some unit tests and it would be great if the CI could run them!

Reply:
Currently when i run `npm test --silent` on my local windows machine, there are some cases failed. I'm not sure whether it's an environment issue or cases issue? So disable it first.

Reply:
Agree then, put it on hold.  It runs okay (for me!) via the VS Code "Launch Tests" debug config, but if the command line doesn't work then you're right, let's leave commented out in this PR.